politeiad/cache/cockroachdb/decred.go

```diff
@@ -45,6 +46,9 @@ type decred struct {
 	recordsdb *gorm.DB              // Database context
 	version   string                // Version of decred cache plugin
 	settings  []cache.PluginSetting // Plugin settings
+
+	bbMtx         sync.RWMutex
+	lastBestBlock uint64 // Last best block used to update tables
```

Comment:
The name best block already implies that its the most recent block.  That's what the "best" part means.

```
 	bestBlock uint64 // Last best block used to update tables
```